### DISCUSSION TITLE
Dumb question - Remix terminal not showing Etherscan URL

### DISCUSSION THREAD

**Discussion by crisie93** [smartcontractkit/full-blockchain-solidity-course-py#348] [Q&A] [answered]
Hi! A dumb question about a small thing.  I see on the tutorials that the Remix terminal returns an Etherscan URL after you deploy or interact with a contract. Somehow mine doesn't (so I have been copy pasting the addresses and searching manually - which takes me 5 more seconds but annoys me) I was playing around with the settings but I can't find a way to make it show the URL. Do you know how to do it?

**Comment by cromewar** [comment, marked as answer]
Hello @crisie93 that's strange, did you scroll down and see if you are missing some part of the output? Anyways you can access easily the transactions on your metamask using the blue arrow:

![image](https://user-images.githubusercontent.com/20199180/141533518-a0405b4b-a33f-427e-9ceb-f386a372990b.png)

**Reply by crisie93** [reply]
Thank you so much for the metamask tip! RE: missing some output, I don't think so - but maybe I'm not seeing something.
See screenshot below of what output looks like for me (an example after deploying/transacting a couple of things):

![image](https://user-images.githubusercontent.com/48460745/141538117-97517816-9705-4d19-9afb-71ac76690937.png)

**Reply by cromewar** [reply]
You should click the arrow to see all the output ;)

![image](https://user-images.githubusercontent.com/20199180/141538418-f5c5b1e0-7f3b-44c2-ad2d-8370599a4564.png)

Best regards, Cromewar.

**Reply by crisie93** [reply]
Oh yes! I had tried that before too - but I still didn't see the URL?
![image](https://user-images.githubusercontent.com/48460745/141550212-8ca7c1fb-5076-4b74-b12a-38e6b28a80b5.png)

**Reply by crisie93** [reply]
Sorry if this is very naive - I'm 100% a newbie

**Reply by cromewar** [reply]
Don't worry about it, all of us are new on this technologies. That's strange as it should give you the link. For now use the metamask method I'll try to find more info on this.
